lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1145,10 +1145,11 @@ def _create_deployment_parameters(
             node_context.vm_name = azure_node_runbook.name
             # ssh related information will be filled back once vm is created. If
             # it's Windows, fill in the password always. If it's Linux, the
-            # private key has higher priority.
+            # private key has higher priority to connect a VM. The password is also
+            # filled in the node context if it is defined in runbook
```

Comment:
This comment should be closer to the related line of code.

Reply:
Drop this commit.